### PR TITLE
change references from green-brown to increasing-decreasing

### LINKED ISSUE #81: use `increasing_or_decreasing` instead of `green_or_brown`
`green_or_brown` is being deprecated in `r2dii.analysis`

### PR DESCRIPTION
**Pull request by jacobvjk**
closes #81

### CODE REVIEW

**Comment by jacobvjk** [comment]
relates to https://github.com/RMI-PACTA/workflow.aggregate.loanbooks/pull/39

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# pacta.aggregate.loanbook.plots (development version)
+
+* Added a `NEWS.md` file to track changes to the package.
+* Replaced references to green/brown with references to increasing/decreasing,
+in line with changes in `r2dii.data` and `r2dii.analysis`
+
+# pacta.aggregate.loanbook.plots 0.0.0.9000
```
Comment (cjyetman) [inline NEWS.md:7]: The current (dev) version on main appears to be 0.0.0.9000, so it's not clear to me why these news items are separated into different sections

**Review by cjyetman — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (cjyetman) [reply NEWS.md:7]: 🤷🏻

Comment (jacobvjk) [reply NEWS.md:7]: for now I incremented the package version to 0.0.0.9001 based on https://combine-australia.github.io/r-pkg-dev/versioning.html

**Pull request merged**